translator/sample files/navjoy_data/navjoy_generated_work_zone_wzdx.json

```diff
@@ -0,0 +1,2054 @@
+{
```

Comment:
Is this just an example from the navjoy self-generated wzdx message set? Are we using it?

Reply:
This is an example. We are not using it, but it could be nice to reference in the future

Reply:
Would it make sense to add a section in the README about NavJoy and how we are translating the 568 form, but they already create a WZDx message from their other form (an example is provided...)?

Reply:
Added some text in the NavJoy translator section with a link to the sample files folder